### PR TITLE
Deployment for dist2src-update listener

### PR DESCRIPTION
**Pull request by lbarcziova**

### CODE REVIEW

File: deployment/roles/deploy/tasks/deploy-centosmsg.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+---
+- name: Deploy centosmsg
+  k8s:
+    namespace: "{{ project }}"
+    definition: "{{ lookup('template', '../templates/dist2src-centosmsg.yml.j2')  }}"
```
Comment (jpopelka) [inline deployment/roles/deploy/tasks/deploy-centosmsg.yml:8]: I tried in #82 and the template lookup plugin finds the file in template/

```suggestion
    definition: "{{ lookup('template', 'dist2src-centosmsg.yml.j2')  }}"
```

**Review by jpopelka — COMMENTED** [review]
Do you prefer me merging the #82 as is now so that we both have a common base layout/structure to work on?
It's unfinished, but It works in local `oc cluster up`. 

Or do you prefer me polishing it first until I'm sure what the layout/structure should look like?

**Comment by lbarcziova** [comment]
@jpopelka if you plan to make some bigger changes in structure, I can wait, otherwise this could be merged if you do not mind.

**Comment by jpopelka** [comment]
#82 will be zuul-merged soon. I don't plan any big changes in the structure since a role directory layout is [rigidly defined](https://docs.ansible.com/ansible/latest/user_guide/playbooks_reuse_roles.html). If you see anything that could be improved in the layout I'll be glad if you raise your voice and let me know so that it could be improved. Thanks

File: deployment/roles/deploy/tasks/deploy-centosmsg.yml (hunk 1 above)
Comment (jpopelka) [reply deployment/roles/deploy/tasks/deploy-centosmsg.yml:8]: I found an even more elegant way, check the `with_template:` in my [tasks/deploy-worker.yml](https://github.com/packit/dist-git-to-source-git/pull/82/files#diff-6778ef9bd59ff7935ac0ff54081b85bd)

File: deployment/roles/deploy/templates/centos-secrets.yml.j2 [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: centos-secrets
+type: Opaque
+data:
+  centos-server-ca.cert: "{{ lookup('file', '{{ path_to_secrets }}/centos-server-ca.cert') | b64encode }}"
+  centos.cert: "{{ lookup('file', '{{ path_to_secrets }}/centos.cert') | b64encode }}"
```
Comment (jpopelka) [inline deployment/roles/deploy/templates/centos-secrets.yml.j2:12]: I'd start simple, with the path to secrets hardcoded.

```suggestion
  centos-server-ca.cert: "{{ lookup('file', 'secrets/centos-server-ca.cert') | b64encode }}"
  centos.cert: "{{ lookup('file', 'secrets/centos.cert') | b64encode }}"
```

I tried locally and it works if you put those `*.cert` into `roles/deploy/files/secrets/`.

Comment (jpopelka) [reply deployment/roles/deploy/templates/centos-secrets.yml.j2:12]: I've created #87 which you can build upon.

**Comment by jpopelka** [comment]
LGTM, is it still WIP?

**Comment by lbarcziova** [comment]
> LGTM, is it still WIP?

I want to verify the subtopics we will filter, otherwise, it is not. And since it does not affect anything now and listener has disabled sending the tasks, it is okay to merge.

**Review by jpopelka — APPROVED** [review]

**Comment by lbarcziova** [comment]
recheck

**Pull request merged**